etc/kayobe/environments/ci-multinode/inventory/group_vars/seed/network-interfaces

```diff
@@ -1,13 +1,15 @@
 ---
 ###############################################################################
-# Network interface definitions for the controller group.
+# Network interface definitions for the seed group.
 
-admin_interface: eth0
+admin_oc_interface: "{{ 'ens2' if os_distribution == 'ubuntu' else 'ens3' }}"
+
+provision_oc_interface: "{{ vxlan_interfaces[0].device}}.{{ provision_oc_vlan }}"
+
+external_interface: "{{ vxlan_interfaces[0].device }}.{{ external_vlan }}"
+
+public_interface: "{{ vxlan_interfaces[0].device }}.{{ public_vlan }}"
 
```

Comment:
This seed.yml can found here: https://github.dev/stackhpc/stackhpc-kayobe-config/blob/b23cd49f142e77771995b700836cdf6f27ae0e76/etc/kayobe/environments/ci-multinode/seed.yml#L1